### PR TITLE
Allow to set agenda item tags with agenda_tag_ids

### PR DESCRIPTION
**Pull request by reiterl**
Remove topic/tag_ids because it is not used in the client.
Update topic.create.

### CODE REVIEW

File: tests/system/action/topic/test_create.py [hunk 1]
```diff
@@ -159,47 +167,6 @@ def test_create_multiple_with_existing_sequential_number(self) -> None:
         topic = self.get_model("topic/3")
         self.assertEqual(topic.get("sequential_number"), 44)
 
-    def test_create_meeting_id_tag_ids_mismatch(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/topic/test_create.py:162]: Keep this test and change to `agenda_tag_ids`

File: tests/system/action/topic/test_create.py [hunk 2]
```diff
@@ -159,47 +167,6 @@ def test_create_multiple_with_existing_sequential_number(self) -> None:
...
-        response = self.request(
-            "topic.create", {"meeting_id": 1, "title": "A", "tag_ids": [8]}
-        )
-        self.assert_status_code(response, 400)
-        assert (
-            "The following models do not belong to meeting 1: ['tag/8']"
-            in response.json["message"]
-        )
-
-    def test_create_with_tag_ids(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/topic/test_create.py:184]: Keep this test and change to `agenda_tag_ids`

File: tests/system/action/topic/test_update.py
Comment (jsangmeister) [inline tests/system/action/topic/test_update.py:1]: Same for these tests

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: tests/system/action/topic/test_create.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/topic/test_create.py:162]: Change the test_create_meeting_id_tag_ids_mismatch into use of `agenda_tag_ids`.

File: tests/system/action/topic/test_create.py (hunk 2 above)
Comment (reiterl) [reply tests/system/action/topic/test_create.py:184]: Change test_create_with_tag_ids into agenda_tag_ids.

File: tests/system/action/topic/test_update.py
Comment (reiterl) [reply tests/system/action/topic/test_update.py:1]: The topic.update action doesn't have set tag_ids anymore. So these tests are not needed anymore.

Comment (jsangmeister) [reply tests/system/action/topic/test_update.py:1]: Then please transfer these tests to the agenda item update tests. They do not only test the action, but also the relation and action handling and should therefore not be removed.

Comment (reiterl) [reply tests/system/action/topic/test_update.py:1]: Okay, I have moved the removed tag_ids update to agenda_item.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**